src/components/lib/dpdk/Makefile

```diff
@@ -51,23 +51,22 @@ MUSL_BIN_PATH = PATH=$(shell pwd)/../libc/musl-1.2.0/bin/:$(PATH)
 config: 
 	$(info Configing DPDK)
 #	TODO: make this copy more generic so that it can be applied to different OS platforms like Fedora
-	@mkdir -p $(DPDK_INC_DIR);
+	mkdir -p $(DPDK_INC_DIR);
```

Comment:
I think it should be that @evanstella wants to know every usefuly compiling command when compiling. I intentionally slient some of them in DPDK because I think they are not that important. I'm OK with either change.

Reply:
Yeah I got some errors during compilation that I wasnt able to debug with the muted output

Reply:
> Any changes here might interfere with @betahxy 's DPDK stuff. I'll let him comment on any potential issues here.

Yeah I got some errors during compilation that I wasnt able to debug with the muted output.